src/xbps/troubleshooting/common-issues.md

```diff
@@ -1,4 +1,18 @@
-# Common Errors
+# Common Issues
+
+## Verifying RSA keys
+
+If you are installing Void for the first time or the Void RSA key has changed,
+[xbps-install(1)](https://man.voidlinux.org/xbps-install.1) might report:
+
+```
+<REPO> repository has been RSA signed by <RSA-FINGERPRINT>
```

Comment:
Shouldn't placeholders use `_` ?

Reply:
Indeed. i've also made some other style changes in that area. i'll create a PR to update the style guide to mention that placeholders should be lower-case, since that's what we're usually using elsewhere (TTBOMK).

Reply:
Oh wait, i've already included that. %-} Obviously still not with it yet.

Reply:
Heh, it happens :P